include/CppUTestExt/MockCheckedExpectedCall.h

```diff
@@ -36,10 +36,12 @@ class MockCheckedExpectedCall : public MockExpectedCall
 
 public:
     MockCheckedExpectedCall();
+    MockCheckedExpectedCall(unsigned int numCalls);
     virtual ~MockCheckedExpectedCall();
 
     virtual MockExpectedCall& withName(const SimpleString& name) _override;
-    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override;
+    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override { return withCallOrder(callOrder, callOrder); }
```

Comment:
@jgonzalezdr I think it's okay to leave it until the method is removed. There were worse problems for the Ti Cl2000 in the past and the only person who noticed them was me :)